holmes/plugins/toolsets/azure_sql/tools/get_top_log_io_queries.py

```
@@ -0,0 +1,189 @@
+)
+from holmes.plugins.toolsets.azure_sql.apis.azure_sql_api import AzureSQLAPIClient
+
+
+def _format_timing(microseconds: float) -> str:
+    """Format timing values with appropriate units (seconds, milliseconds, microseconds)."""
+    if microseconds >= 1_000_000:  # >= 1 second
+        return f"{microseconds / 1_000_000:.2f} s"
+    elif microseconds >= 1_000:  # >= 1 millisecond
+        return f"{microseconds / 1_000:.2f} ms"
+    else:  # < 1 millisecond
+        return f"{microseconds:.0f} μs"
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)